### DISCUSSION TITLE
Configuration for "at least one" of the images

### DISCUSSION THREAD

**Discussion by roderik** [nftchef/art-engine#183] [Help + Q&A]
I'm trying to get the generator to pick at least one of the images for some of the layers. But not sure how to do it.

```
layers
 - Background
   - Blue#25.png
   - Green#25.png
   - Orange#25.png
   - Yellow#25.png
 - Other
   - some optional background overlays
 - Base
   - Character.png 
 - OnTop
   - other optional overlays
```

```
const layerConfigurations = [
  {
    growEditionSizeTo: 100,
    namePrefix: "XXX", // Use to add a name to Metadata `name:`
    layersOrder: [
      { name: "Background" },
      { name: "Other" },
      { name: "Base" },
      { name: "OnTop" },
    ]
  },
``` 

As I understand the docs

- for background there is a 25% chance a color gets picked. But I also get "white" backgrounds. Removing the rarities adds them all to the image at the same time

Repo's latest commit: https://github.com/nftchef/art-engine/commit/3b77a298c937a0c56109034a140bda0ca9671266

**Comment by BakeSloth** [comment]
make sure you dont have a blank png for background. or transparent backgrounds. 
The script stacks pngs on top of each other, so if you have a blank or transparent background, it will cause issues similar to what you are experiencing if I read correctly.

all backgrounds should be solid.
